### DISCUSSION TITLE
Trying to get Xenia to run, but so far no luck...

### DISCUSSION THREAD

**Discussion by cyberwolfie1** [bottlesdevs/Bottles#3614] [General]
I have spent several hours now trying to get Xenia to run with Bottles. I had it running at the start of the day through EmuDeck, and could play one of my old Xbox 360 games on it, but I really didn't like EmuDeck and thought I could get it running myself and uninstall EmuDeck. But I've had no luck so far, so I'm posting here as a last resort. 

I tried going through their bash scripts to find how they did it, and they seem to invoke Proton directly from the Steam-directory. I am not used to reading bash-scripts though, and I got lost in exactly how they call it, and I think I need something like Bottles to help me get things right...

I have Bottles 51.17 installed through Flatpak. First of, I can't launch it unless I run `GSK_RENDERER=ngl flatpak run com.usebottles.bottles` in the Terminal. Otherwise it just crashes - is there a way to solve this more permanent? I am using Wayland. I also get tons of warnings from the Terminal window I am running: `(bottles:2): Gdk-WARNING **: 22:35:17.934: ../gdk/wayland/gdkcursor-wayland.c:210 cursor image size (96) not an integer multiple of theme size (36)`. 

Ok, so to the actual issues I am facing:

I have made a Bottle to launch Xenia (or rather, Xenia Canary) through. I have tried the following runners:
- `ge-proton9-21`
- `wine-ge-proton8-26`
- `soda-9.0-1`
- `sys-wine-9.0`

My guess is I want to use either `ge-proton9-21` or `wine-ge-proton8-26`. Under DXVK I have `dxvk-2.5.1` and under VKD3D I have `vkd3d-proton-2.14`.

I have via Flatseal given access to my local Steam-folder, so under "Compatability" I have checked "Steam Runtime". It seems I am still missing some libraries though. If I try to run `xenia_canary.exe` with `ge-proton9-21` I get the following errors:

``` log
002c:err:wineboot:process_run_key Error running cmd L"C:\\windows\\system32\\winemenubuilder.exe -a -r" (126).
0024:err:module:import_dll Library libvkd3d-1.dll (which is needed by L"C:\\windows\\system32\\wined3d.dll") not found
0024:err:module:import_dll Library libvkd3d-shader-1.dll (which is needed by L"C:\\windows\\system32\\wined3d.dll") not found
0024:err:module:import_dll Library wined3d.dll (which is needed by L"C:\\windows\\system32\\dxgi.dll") not found
0024:err:module:import_dll Library dxgi.dll (which is needed by L"Z:\\run\\user\\1000\\doc\\7738baa2\\xenia_canary.exe") not found
0024:err:module:loader_init Importing dlls for L"Z:\\run\\user\\1000\\doc\\7738baa2\\xenia_canary.exe" failed, status c0000135
```

If I instead use `wine-ge-proton8-26`, I instead get these:

```
002c:err:wineboot:process_run_key Error running cmd L"C:\\windows\\system32\\winemenubuilder.exe -a -r" (126).
008c:err:hid:udev_bus_init UDEV monitor creation failed
0140:err:seh:check_noexcept noexcept function propagating exception
```
Sometimes the program will launch for a moment before crashing. The `022c:err:wineboot:` line will also show up when changing runner, not just when trying to launch the .exe.

Bottom line is, I have no clue where to go from here. This is the first time I am doing anything with Wine/Proton outside of Steam or Lutris. 

Is there some obvious thing I am doing wrong here?
